configs/rk3588-axon_defconfig

```diff
@@ -34,7 +34,7 @@ CONFIG_SPL_FIT_IMAGE_KB=2048
 CONFIG_BOOTDELAY=1
 CONFIG_SYS_CONSOLE_INFO_QUIET=y
 # CONFIG_DISPLAY_CPUINFO is not set
-CONFIG_ANDROID_BOOTLOADER=y
+# CONFIG_ANDROID_BOOT_IMAGE is not set
 CONFIG_ANDROID_AVB=y
```

Comment:
remove every android flag and try once

Reply:
@Pratiksha0102 Could you remove all android flags which are not required to build Linux Image from rk3588-axon_defconfig file and build Full Ubuntu Mate Kernel 5.10 Image ?